tests/phpunit/tests/option/wpSetOptionAutoload.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Test wp_set_option_autoload().
+ *
+ * @group option
+ *
+ * @covers ::wp_set_option_autoload
+ */
+class Tests_Option_WpSetOptionAutoload extends WP_UnitTestCase {
+
+	/**
+	 * Tests that setting an option's autoload value to 'yes' works as expected.
+	 *
+	 * @ticket 58964
+	 */
+	public function test_wp_set_option_autoload_yes() {
```

Comment:
Can we add additional test cases for boolean values?

Reply:
Added in https://github.com/WordPress/wordpress-develop/pull/5069/commits/bbe09133fd947c852434a06b3007a2665cc00ae7